packages/compass-collection/src/index.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { expect } from 'chai';
```

Comment:
Can’t comment on `packages/compass-collection/src/index.d.ts`, so I’ll do it here – it’s an empty file that should not be necessary, should we just drop it?

Reply:
Thanks for noticing it! Added it temporarily and forgot to delete ✨